### PR TITLE
SDK-2151 Static Liveness Check

### PR DESCRIPTION
**Pull request by LuborRod**

### CODE REVIEW

**Review by BenSmithYoti — APPROVED** [review]
LGTM and yes this seems to match the spec in Jira ticket. Cannot run the code easily here, but I have reviewed on Github and (from my understanding) this is all correct and ready to go. Yes. Thanks

File: tests/DocScan/Session/Retrieve/StaticLivenessResourceResponseTest.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                ]
+            ],
+
+        ];
+
+        $result = new StaticLivenessResourceResponse($input);
+
+        $this->assertEquals(self::SOME_LIVENESS_TYPE, $result->getLivenessType());
+        $this->assertNotNull($result->getImage());
+        $this->assertInstanceOf(MediaResponse::class, $result->getImage());
```
Comment (irotech) [inline tests/DocScan/Session/Retrieve/StaticLivenessResourceResponseTest.php:41]: shall we also assert the security id stored inside the media?

Comment (LuborRod) [reply tests/DocScan/Session/Retrieve/StaticLivenessResourceResponseTest.php:41]: Yes, I will add now.

**Review by irotech — APPROVED** [review]
Def not a PHP dev and I reviewed on GitHub without checking out and run the code. LGTM

**Pull request merged**